docs/source/examples/combine_pt_ddd_zne_rem.md

```diff
@@ -0,0 +1,235 @@
+    bitstrings = result.measurements['m']
+    
+    # Compute expectation of Z0 * Z1 = parity of first two qubits
+    values = [(-1) ** (b[0] ^ b[1]) for b in bitstrings]
+    expectation = sum(values) / len(values)
+    
+    return expectation
+```
+```{code-cell} ipython3
+from mitiq.ddd import execute_with_ddd, rules
+
+#No observables
+
+ddd_result = execute_with_ddd(
+    circuit=twirled_ghz,
+    executor=executor,
+    observable=None,  
+    rule=rules.xx,
+    num_trials=1,
+    full_output=True
+)
+
+```
+```{code-cell} ipython3
+ddd_result
+```
+```
+(-1.0,
+ {'ddd_value': -1.0,
+  'ddd_trials': [-1.0],
+  'circuits_with_ddd': [0: ───H───Y───@───Y───I───X───X───
+              │
+1: ───X───────X───X───I───@───I───
+                          │
+2: ───X───I───X───X───I───X───X───]})
+```
+```{code-cell} ipython3
+ddd_circuit = ddd_result[1]['circuits_with_ddd'][0]
+ddd_circuit.append(cirq.measure(*ddd_circuit.all_qubits(), key='m'))
+```
+```
+ddd_circuit
+```
+```
+0: ───H───Y───@───Y───I───X───X───M('m')───
+              │                   │
+1: ───X───────X───X───I───@───I───M────────
+                          │       │
+2: ───X───I───X───X───I───X───X───M────────
+```
+
+## Combining Readout Error Mitigation (REM) and Zero Noise Extrapolation (ZNE)
+
+```{code-cell} ipython3
+def execute(circuit: cirq.Circuit, noise_level: float = 0.002, p0: float = 0.05) -> MeasurementResult:
+    measurements = circuit[-1]
+    circuit = circuit[:-1]
+    circuit = circuit.with_noise(cirq.depolarize(noise_level))
+    circuit.append(cirq.bit_flip(p0).on_each(circuit.all_qubits()))
+    circuit.append(measurements)
+
+    simulator = cirq.DensityMatrixSimulator()
+    result = simulator.run(circuit, repetitions=10000)
+
+    # Extract bitstrings in correct order
+    measurement_key = list(result.measurements.keys())[0]  # e.g., "m"
+    bitstrings = result.measurements[measurement_key]
+    return MeasurementResult(bitstrings)
+```
+
+```{code-cell} ipython3
+qubits = [cirq.LineQubit(i) for i in range(3)]
+cirq_ps = cirq.PauliString({qubits[0]: cirq.Z, qubits[2]: cirq.Z})  # Note: qubit 1 with identity omitted
+
+spec = "".join(str(cirq_ps[q]) if q in cirq_ps else 'I' for q in qubits)
+support = tuple(range(len(qubits)))
+
+obs = Observable(PauliString(spec, support=support))
+print(obs)
+```
+```{code-cell} ipython3
+noisy = raw.execute(ddd_circuit, execute, obs)
+```
+```{code-cell} ipython3
+from functools import partial
+
+ideal = raw.execute(ddd_circuit, partial(execute, noise_level=0, p0=0), obs)
+```
+
+```{code-cell} ipython3
+print("Unmitigated value:", "{:.12f}".format(noisy.real))
+```
+
+```
+Unmitigated value: -0.959200000000
+```
+
+```{code-cell} ipython3
+p0 = p1 = 0.05  # Your readout error probabilities
+
+num_measured_qubits = 3
+icm = rem.generate_inverse_confusion_matrix(num_measured_qubits, p0, p1)
+
+rem_executor = rem.mitigate_executor(execute, inverse_confusion_matrix=icm)
```

Comment:
I don't think there are any measurement errors introduced in your noise model, are there?